src/test/java/net/sf/jabref/importer/fileformat/BiblioscapeImporterTestFiles.java

```diff
@@ -0,0 +1,59 @@
+package net.sf.jabref.importer.fileformat;
+
+import net.sf.jabref.*;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class BiblioscapeImporterTestFiles {
+
+    private BiblioscapeImporter bsImporter;
+
+    @Parameter
+    public String fileName;
+
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        bsImporter = new BiblioscapeImporter();
+    }
+
+    @Parameters(name = "{0}")
+    public static Collection<String> fileNames() {
+        return Arrays.asList(new String[] {"BiblioscapeImporterTest0a", "BiblioscapeImporterTest0b",
```

Comment:
Is it possible to give the file names more speaking names. Meaning: The names should be self-explanary. For instance, for ttest `1`, you could use `BakloutiEtAl1996`. Oh, I see that all articles are from him. Maybe, you can state the real difference of the test cases using the file name.